pages/infrastructure_providers-validators/optimize_full_node.md

```diff
@@ -0,0 +1,78 @@
+You need a running, non-validating full node that is connected to a network. 
+
+- If you created a system service for your node by following the instructions on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/set_up_full_node.md), start your node with the following command:
+  ```bash
+  stystemctl start dydxprotocold
+  ```
+
+- To start your node with Cosmovisor or with the `dydxprotocold` binary, you must include the flag `--non-validating-full-node=true`. The flag disables the functionality intended for validator nodes and enables additional logic for reading data. Your CLI may prompt you to configure additional variables in your environment or include them in your command.
+  
+  To start your node with Cosmovisor, run the following command:
+  ```bash
+  cosmovisor run start --non-validating-full-node=true
+  ```
+
+  To start your node with `dydxprotocold`, run the following command:
+  ```bash
+  dydxprotocold run start --non-validating-full-node=true 
+  ```
+
+## Save a List of Trusted Nodes
+Specify a list of healthy, stable nodes that you trust. Your node prioritizes connecting to those nodes, speeding up the process of connecting or re-connecting to the network. Connecting directly with a peer node is faster than connecting to a seed node and then finding new peers.
+
+### Save a List of Persistent Peers
+You can save a list of healthy, stable nodes in the `persistent_peers` field of your `config.toml` file.
+
+Request a list of healthy peers for your deployment from a [Live Peer Node](../infrastructure_providers-network/resources.mdx#live-peer-node-providers) provider.
+
+From the list of healthy peers that you retrieve from peer node provider, choose any 5 for your node to query for the latest state. Add a comma-separated list of those peer addresses to the `persistent_peers` field in your `config.toml`, like in the following example:
+
+```yaml
+# config.toml
+# Example values from Polkachu for dydx-mainnet-1
+persistent_peers=83c299de2052db247f08422b6592e1383dd7a104@136.243.36.60:23856,1c64b35055d34ff3dd199bb4a5a3ae46b9c10c89@3.114.126.71:26656,3651c82a89f8f4d6fc30fb27b91159f0de092031@202.8.9.134:26656,580ec248de1f41d4e50abe132b7838348db55b80@176.9.144.40:23856,febe75fb6e70a60ce6344b82ff14903bcb53a209@38.122.229.90:26656
+```
+
+### Replace Your Address Book File
+As an alternative to persistent peers, you can replace your node's local address book with the latest address book from a trusted provider. The address book file contains the latest connection information for peers from that provider.
+
+Download an up-to-date `addrbook.json` file for your deployment from an [Address Book](../infrastructure_providers-network/resources.mdx#address-book-providers) provider. 
+
+Save it in your `/.dydxprotocol/config` directory, replacing the existing `addrbook.json` file.
+
+## Prepare to Restore Your Node
+Prepare to restore your node quickly in case it falls out of sync, minimizing downtime.
```

Comment:
nit / personal preference: will defer to you
```suggestion
To minimize downtown, prepare to restore your node quickly in case it falls out of sync.
```

Reply:
Updating to:

To minimize downtime in case your node falls out of sync, make preparations to restore your node quickly. 

Starting with "To minimize downtime, ..." and ending with "in case it falls out of sync" requires the reader to hold context in mind til the end of the sentence to understand that we're minimizing "out of sync" downtime. Until the end of the sentence, a reader doesn't know what downtime we're talking about